kinesumer.go

```diff
@@ -54,19 +57,20 @@ var DefaultOptions = Options{
 	ShardAcquisitionTimeout: 90 * time.Second,
 }
 
-func NewDefault(stream string) (*Kinesumer, error) {
+func NewDefault(stream, duration string) (*Kinesumer, error) {
 	return New(
 		kinesis.New(session.New()),
 		nil,
 		nil,
 		nil,
 		stream,
 		nil,
+		duration,
 	)
 }
 
 func New(kinesis k.Kinesis, checkpointer k.Checkpointer, provisioner k.Provisioner,
-	randSource rand.Source, stream string, opt *Options) (*Kinesumer, error) {
+	randSource rand.Source, stream string, opt *Options, duration string) (*Kinesumer, error) {
```

Comment:
Yea, I initially thought the logic for parsing the duration was going to be more complicated so wanted to handle that in the library.

The advantage of it being a string is that `emp log` and `kinesumer` can pass the same value, otherwise both are implementing the `time.ParseDuration`.